OKTNav/OKTNav/client/src/main/java/cs321/group1/oktnav/Room.java

```diff
@@ -5,10 +5,40 @@
  */
 package cs321.group1.oktnav;
 
+import java.util.ArrayList;
+
 /**
  *
  * @author Tobin
```

Comment:
You can remove this comment block if you want; such that the Javadoc doesn't get confused.